File: java/050924_B4_11283.java
/*
 * https://www.acmicpc.net/problem/11283
 */
import java.io.*;

class B4_11283 {
	public static void main(String[] args) throws IOException {
		try(var br = new BufferedReader(new InputStreamReader(System.in))) {
			int line = br.read();
			System.out.println(line-'가'+1);
		}
	}
}

File: go/050924_B4_11283.go
/*
 * https://www.acmicpc.net/problem/11283
 */
 package main

 import (
	 "fmt"
	 "bufio"
	 "os"
 )

 func main() {
	 rw := bufio.NewReadWriter(bufio.NewReader(os.Stdin), bufio.NewWriter(os.Stdout))
	 defer rw.Flush()
	
	 var c rune
	 fmt.Fscanf(rw, "%c", &c)
	 fmt.Fprintf(rw, "%d\n", int(c)-int('가')+1)
 }
